Week-2/Exercise2.1/index.scss

```diff
@@ -0,0 +1,331 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 16px;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+/*16px*/
+
+body {
+  background: white;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  line-height: 1.75;
+  color: #1e1e1e;
+}
+
+p {
+  margin-bottom: 1rem;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  margin: 3rem 0 1.38rem;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  line-height: 1.3;
+}
+*::selection {
+  background-color: #1e1e1e7a;
+}
+
+h1 {
+  font-size: 3.052rem;
+}
+
+h2 {
+  font-size: 2.441rem;
+}
+
+h3 {
+  font-size: 1.953rem;
+}
+
+h4 {
+  font-size: 1.563rem;
+}
+
+h5 {
+  font-size: 1.25rem;
+}
+
+small,
+.text_small {
+  font-size: 0.8rem;
+  display: block;
+  margin-bottom: 1rem;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+img {
+  user-select: none;
+  -moz-user-select: none;
+  -webkit-user-drag: none;
+  -webkit-user-select: none;
+  -ms-user-select: none;
+}
+
+header {
+  width: 100vw;
+  padding: 1rem 5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 1px 1px 10px #000;
+
+  .navLogoBox {
+    padding: 0.85rem;
+    background-color: #ff6e6e;
+    transition: 0.3s ease-in-out;
+    -webkit-transition: 0.3s ease-in-out;
+    -moz-transition: 0.3s ease-in-out;
+    -ms-transition: 0.3s ease-in-out;
+    -o-transition: 0.3s ease-in-out;
+    border-radius: 1rem;
+    -webkit-border-radius: 1rem;
+    -moz-border-radius: 1rem;
+    -ms-border-radius: 1rem;
+    -o-border-radius: 1rem;
+
+    &:hover {
+      background-color: #1e1e1e;
+      color: #dcdcdc;
+    }
+
+    h4 {
+      margin: 0;
+      // font-weight: 500;
+      font-family: "Dancing Script", cursive;
+    }
+  }
+
+  ul {
+    list-style: none;
+    width: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+
+    li {
+      a {
+        font-size: 1.25rem;
+        font-weight: 500;
+        letter-spacing: 1.5px;
+        position: relative;
+
+        &::after {
+          width: 0;
+          position: absolute;
+          content: "";
+          left: 0;
+          bottom: -30%;
+          height: 0.4vh;
+          background-color: rgb(255, 110, 110);
+          transition: 0.5s ease-in-out;
+          -webkit-transition: 0.5s ease-in-out;
+          -moz-transition: 0.5s ease-in-out;
+          -ms-transition: 0.5s ease-in-out;
+          -o-transition: 0.5s ease-in-out;
+        }
+
+        &:hover::after {
+          width: 100%;
+        }
+      }
+    }
+  }
+}
+
+main {
+  width: 100vw;
+  padding: 0 7rem 2rem;
+
+  .title {
+    font-size: 8rem;
+  }
+
+  .featuredSection {
+    width: 100%;
+    min-height: 50vh;
+    padding: 2rem 0;
+    display: flex;
+    gap: 2.5rem;
+
+    .featured,
+    .articles {
+      height: 100%;
+    }
+
```

Comment:
Good use of scss to writing nested styles to maintain the precedence order. keep it up.